client/tdh/customer-metadata/service.go

```diff
@@ -46,14 +46,18 @@ func (s *Service) GetPolicies(query *PoliciesQuery) (model.Paged[model.Policy],
 }
 
 // GetUsers - Return list of Users
-func (s *Service) GetUsers(query *UsersQuery) (model.Paged[model.User], error) {
+func (s *Service) GetUsers(query *UsersQuery, isSRE bool) (model.Paged[model.User], error) {
```

Comment:
it is not required, just use `s.Api.IsSRE` inside the method, that's why we added, for global access from inside client/service